src/screens/Home/styles.ts

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+import AwesomeAlert from "react-native-awesome-alerts";
+
+import { scale } from "../../utils/scalling";
+
+export const OccurrenceAlert = styled(AwesomeAlert).attrs({
```

Comment:
Why did u create another component with the same styles as StayAlert.js? (from src/components). Isn't it better to just reuse it or rename it?

Reply:
Thanks, @alvesitalo! I create a StayAlert component now for use on all screens that needs alert